### DISCUSSION TITLE
is it possible for code in `buildSrc/src/main/java` to reference and use code from `app/java/de/westnordost/streetcomplete/osm`?

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4422] [Q&A] [answered]
Is there any plausible way to for example reference `IS_SHOP_OR_DISUSED_SHOP_EXPRESSION` without redefining, within script residing in `buildSrc`?

And why I want this? See for example https://github.com/matkoniecz/Zazolc/blob/taginfo/buildSrc/src/main/java/UpdateTaginfoListingTask.kt#L58

And if #4419 gets merged then much bigger function will become duplicated - so I would want to start reusing it. If possible.

**Comment by westnordost** [comment, marked as answer]
No
